### PR TITLE
Update to 1.20.2

### LINKED ISSUE #43: Incompatible with 1.20.2
Attempting to start up the game with latest mod causes the client to [crash](https://pastebin.com/raw/u0zRmUp1).

### PR DESCRIPTION
**Pull request by TechPro424**
This PR updates the mod to 1.20.2, fixes #43 within the mod itself, rather than having to download a fork

### CODE REVIEW

**Comment by ghost** [comment]
@TechPro424 
Im stuck here. What do i download to make it work? Im not that much into github knowledge.

**Comment by TechPro424** [comment]
The mod author has to merge my PR before you can download it from their official download pages. However, if you want to use this version right now, here's the mod jar (this is my unofficial version)

[horse-stats-vanilla-4.5.0.zip](https://github.com/d4rkm0nkey/HorseStatsVanilla/files/13407577/horse-stats-vanilla-4.5.0.zip)

Extract the jar file from the zip file and place the jar file in your mods folder

Sorry for the late response, was sick for the last few days

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   "depends": {
     "fabricloader": ">=0.14.21",
     "java": ">=17",
-    "minecraft": ">=1.20",
+    "minecraft": ">=1.20.2",
```
Comment (leopoldhub) [inline src/main/resources/fabric.mod.json:33]: the minecraft dependency was `1.20` for the `1.20` and `1.20.1` version so we may want to leave it to `1.20` for all `1.20.*` versions?

**Review by leopoldhub — APPROVED** [review]
I am not sure about the Minecraft dependency jump (`>=1.20` => `>=1.20.2`) (see related comment), but the other changes seem ok to me.

**Pull request merged**